### DISCUSSION TITLE
Support for Ceph Object Storage Gateway

### DISCUSSION THREAD

**Discussion by axoxymous-coder** [nginxinc/nginx-s3-gateway#129] [Q&A]
Hi,

Is nginx-s3-gateway supported with Ceph Object Storage Gateway? I tried using it with Ceph, but it always errors out with AWS EC2 credentials not found. My Ceph cluster is created using Rook in local K8s cluster. 
Here are the settings I am using for Ceph:

```
S3_BUCKET_NAME=mybucket
AWS_ACCESS_KEY_ID=XXXXXXXXXXXXXXXXXX
AWS_SECRET_KEY=YYYYYYYYYYYYYYYYYYYYYYYYYYYYY
S3_SERVER=172.17.0.1
S3_SERVER_PORT=31185
S3_SERVER_PROTO=http
S3_REGION=us-east-1
AWS_SIGS_VERSION=4
ALLOW_DIRECTORY_LIST=true
S3_STYLE=virtual
DEBUG=true
PROVIDE_INDEX_PAGE=false
APPEND_SLASH_FOR_POSSIBLE_DIRECTORY=true
PROXY_CACHE_VALID_OK=1h
PROXY_CACHE_VALID_NOTFOUND=1m
PROXY_CACHE_VALID_FORBIDDEN=30s
```

Any help will be highly appreciated.

**Comment by dekobon** [comment]
See discussion in issue: https://github.com/nginxinc/nginx-s3-gateway/issues/130
